datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -852,8 +852,8 @@ private void checkIfSdxIsDeletable(SdxCluster sdxCluster, boolean forced) {
                 throw ex;
             }
         }
-        if (attachedDistroXClusters != null && !attachedDistroXClusters.isEmpty()) {
-            throw new BadRequestException(String.format("The following Data Hub cluster(s) must be terminated before SDX deletion [%s]",
+        if (!forced && attachedDistroXClusters != null && !attachedDistroXClusters.isEmpty()) {
+            throw new BadRequestException(String.format("The following Data Hub cluster(s) must be terminated before SDX deletion [%s]. Please use force delete to override this check.",
```

Comment:
You mention in the JIRA, "We can check if the datahub is stopped, but we shouldn't block this operation." I think that'd be a good thing to check. Could we add that?

Reply:
I agree, we should check if the data hubs are stopped even when if it a forcible deletion.

Reply:
Ok i'll add the check. So basically you can force this ONLY if the datahubs are stopped, right? Does that sound like the correct logic?

Reply:
You might have situations where you can't stop datahubs due to a failure. I have a cluster where I can't stop FreeIPA and I have seen a datalake that couldn't be stopped too.